packages/react/src/lib/accessibility/Behaviors/Menu/menuItemBehavior.ts

```diff
@@ -45,8 +45,6 @@ const menuItemBehavior: Accessibility = (props: any) => ({
         ? true
         : undefined,
       [IS_FOCUSABLE_ATTRIBUTE]: !props['disabled'],
-      'aria-posinset': props.itemPosition,
```

Comment:
Should we remove this props in component too? I.e. fully revert #1066?

Reply:
no, we would like still to have the possibility for the users to pick them up if they want